Modules/+Experiments/+AutoExperiment/@AutoExperiment_invisible/run.m

```diff
@@ -34,6 +39,7 @@ function run(obj,status,managers,ax)
         end
     end
 else % Start a new experiment
+    assert(isempty(obj.analysis),'Analysis loaded. Did you mean to continue an experiment?')
```

Comment:
Do we want this to be a straight up error? Or if the answer is "yes, I meant to continue an experiment", should there be an option to do that?

Reply:
Not that many more buttons to go click "continue" and "run" again - will re-enforce the habit next time.
If it isn't a huge game changer to include extra logic like you're suggesting it usually isn't worth the additional complexity.